mono/mini/mini-runtime.h

```diff
@@ -422,10 +422,11 @@ MONO_API void        mono_parse_env_options         (int *ref_argc, char **ref_a
 MONO_API char       *mono_parse_options_from        (const char *options, int *ref_argc, char **ref_argv []);
 MONO_API int         mono_regression_test_step      (int verbose_level, const char *image, const char *method_name);
 
-
 void                   mono_interp_stub_init         (void);
 void                   mini_install_interp_callbacks (const MonoEECallbacks *cbs);
 
+extern char           *stats_method_name;
```

Comment:
I would put this right under `sdb_options` with most of the externs. `mono_interp_callbacks_pointer` is only separated to group it with `mini_install_interp_callbacks`, but there's no reason for `stats_method_name` to live here.